index.html

```diff
@@ -14,6 +14,8 @@
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.8/css/materialize.min.css">
 		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 		<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.8/js/materialize.min.js"></script>
+		<meta name="ember-app/config/environment" content="%7B%22modulePrefix%22%3A%22ember-app%22%2C%22environment%22%3A%22development%22%2C%22rootElement%22%3A%22%23ember-app%22%2C%22rootURL%22%3A%22/%22%2C%22locationType%22%3A%22auto%22%2C%22EmberENV%22%3A%7B%22FEATURES%22%3A%7B%7D%2C%22EXTEND_PROTOTYPES%22%3A%7B%22Date%22%3Afalse%7D%7D%2C%22APP%22%3A%7B%22name%22%3A%22ember-app%22%2C%22version%22%3A%220.0.0+918b9135%22%7D%2C%22exportApplicationGlobal%22%3Atrue%7D" />
```

Comment:
according to this : https://stackoverflow.com/questions/26577269/ember-cli-meta-config-environment-file 
we can probably get rid of the meta, and update ember-cli-build.js file instead

let app = new EmberApp(defaults, {
    autoRun: false,
    storeConfigInMeta: false
    // Add options here
  });

to have equivalent effect